plenario_client/client.py

```diff
@@ -0,0 +1,16 @@
+import config
+import requests
+from plenario_response.response import PlenarioResponse
+
+DEFAULT_API_URL = config.API_URL
+
+class PlenarioClient:
+
+    def __init__(self, api_url: str=DEFAULT_API_URL):
+        self.api_url = api_url
```

Comment:
I'm working on the python style guide, so consider this an optional nit pick. I'm a big fan of annotating object attributes set in the `__init__` method.

```python
class PlenarioClient:
    def __init__(self, api_url: str=DEFAULT_API_URL):
        self.api_url: str = api_url
        self.session: Session = request.Session()
        self.response: dict = None
```